src/librados/IoCtxImpl.cc

```diff
@@ -1828,7 +1828,6 @@ int librados::IoCtxImpl::notify(const object_t& oid, bufferlist& bl,
   Context *notify_finish = new C_notify_Finish(client->cct, &notify_finish_cond,
                                                objecter, linger_op, preply_bl,
                                                preply_buf, preply_buf_len);
-  (void) notify_finish;
```

Comment:
@dillaman we might want to keep this to silence the warnings, but probably we could use the `maybe_unused` attribute specifier:
```c++
[[maybe_unused]] auto notify_finish = ...
```